.pre-commit-config.yaml

```
@@ -102,6 +102,22 @@ repos:
       - id: yamllint
         files: ^tmt/schemas/.*\.yaml
 
+  - repo: https://github.com/ansible-community/ansible-lint.git
+    rev: v24.2.0
```

```suggestion
    rev: v24.2.1
```
Some bugfixes and Fedora 40 support. Would be cool to have up-to-date version when merging.
https://github.com/ansible/ansible-lint/releases/tag/v24.2.1

---

Updated :-)